### PR TITLE
My first branch

### PR DESCRIPTION
**Pull request by DannieMo**
Created a Hello Microverse project and implemented git workflow

I update the README file
I added the linting config files

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi @DannieMo , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Add a description to the PR.
- Ensure all linting issues are fixed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

**Comment by mikenath223** [comment]
Hi @DannieMo , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Please add a description to the PR by clicking the three dots icon and selecting update PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @DannieMo, 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- Add all linter config files stated in the linting guide.
- Update the readme to include relevant sections.


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```
Comment (mikenath223) [inline .github/workflows/linters.yml:8]: - [ ] Update the readme to include relevant sections as shown in this [template](https://github.com/microverseinc/readme-template).

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
```
Comment (mikenath223) [inline .github/workflows/linters.yml:28]: - [ ] Ensure you add all linting config files for WebHint and Stylelint as stated in this [link](https://github.com/microverseinc/linters-config/tree/master/html-css). The linting guide helps improve the professionalism of the project and is shared by other students also.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (abruzy) [inline .gitignore]: add ```node_modules/``` as well to the ```.gitignore file```

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @DannieMo 👍🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._
_1. Ensure to add a nice PR title and summary of the changes you made.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Shaher-11 — APPROVED** [review]
#### Hello @DannieMo,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by DannieMo** [comment]
Thank you for your honest reviews.

**Pull request merged**